packages/runtime/container-runtime/src/opLifecycle/outbox.ts

```diff
@@ -101,15 +101,22 @@ export class Outbox {
 		}
 
 		if (++this.mismatchedOpsReported <= this.maxMismatchedOpsToReport) {
-			this.mc.logger.sendErrorEvent(
+			// Increase the stack trace limit temporarily, so as to debug better in case it occurs.
+			const originalStackTraceLimit = (Error as any).stackTraceLimit;
+			(Error as any).stackTraceLimit = 50;
```

Comment:
It's non-standard V8 numerical property, so this will throw in some browsers, I think. Ideally, we create helper that has this usage (and sample implementation):

```ts
const error = getLongStack( () => new UsageError("Submission of an out of order message"));

function getLongStack(action: () => Error) {
   try {
      Error.stackTraceLimit = 50;
      const res = action();
      Error.stackTraceLimit = 10;
      return res;
   } catch (exceptioon) {
      retrun action();
   } finally ()
}
```